### PR TITLE
Remove 4.1 changelogs

### PR DESCRIPTION
**Pull request by dstansby**
# Thank you for contributing to SunPy

## 🚨 IMPORTANT 🚨

We know that working on code and submitting pull requests takes effort, and we appreciate your time.
Thank you.

[Please be aware that everyone has to follow our code of conduct](https://sunpy.org/coc)

[Furthermore, you might need to check with your work place if you are allowed to contribute code](https://docs.sunpy.org/en/latest/dev_guide/contents/newcomers.html)

- Please try to create an issue _before_ creating a Pull Request.
- Please use the following Git commit message style
  - Use the future tense ("Adds feature" not "Added feature")
  - Limit the first line to 72 characters or less

We have several automatic features that run when a pull request is open.
They can appear daunting but do not worry about them!
[This a brief explanation of them.](https://docs.sunpy.org/en/latest/dev_guide/contents/pr_review_procedure.html#continuous-integration)

### CODE REVIEW

**Review by Cadair — APPROVED** [review]
Don't be like Stuart and forget to dev tag main!

**Comment by dstansby** [comment]
> Don't be like Stuart and forget to dev tag main!

I did the opposite and remembered to dev tag when branching, but not to push the deleted changelogs...

**Comment by Cadair** [comment]
That's fine too.

File: changelog/README.rst [hunk 1]
```diff
@@ -1,34 +0,0 @@
-=========
```
Comment (nabobalis) [inline changelog/README.rst:1]: This should stay.

**Comment by nabobalis** [comment]
Do we not update main's changelog with all the 4.1 notes and the bugfixes too or did we do that already?

**Comment by dstansby** [comment]
> Do we not update main's changelog with all the 4.1 notes and the bugfixes too or did we do that already?

I did that already somewhere.

**Pull request closed without merging**